content/docs/security/local-ratelimit.txt

```diff
@@ -0,0 +1,247 @@
+---
+title: Local rate limiting
+weight: 10
+description: Limit the number of requests that are allowed to enter the cluster before global rate limiting and external auth policies are applied.  
+---
+
+To learn more about what local rate limiting is and the differences between local and global rate limiting, see [About local rate limiting](/docs/security/ratelimit/local/).
+
+## About local rate limiting
+
+Local rate limiting is a coarse-grained rate limiting capability that is primarily used as a first line of defense mechanism to limit the number of requests that are forwarded to your rate limit servers. 
+
+Without local rate limiting, all requests are directly forwarded to a rate limit server that you set up where the request is either denied or allowed based on the global rate limiting settings that you configured. However, during an attack, too many requests might be forwarded to your rate limit servers and can cause overload or even failure.
+
+To protect your rate limit servers from being overloaded and to optimize their resource utilization, you can set up local rate limiting in conjunction with global rate limiting. Because local rate limiting is enforced in each Envoy instance that makes up your gateway, no rate limit server is required in this setup. For example, if you have 5 Envoy instances that together represent your gateway, each instance is configured with the limit that you set. 
+
+For more information about local rate limiting, see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/local_rate_limit_filter). 
+
+### Architecture
+
+The following image shows how local rate limiting works in {{< reuse "docs/snippets/product-name.md" >}}. As clients send requests to a backend destination, they first reach the Envoy instance that represents your gateway. Local rate limiting settings are applied to an Envoy pod or process. Note that limits are applied to each pod or process. For example, if you have 5 Envoy instances that are configured with a local rate limit of 10 requests per second, the total number of allowed requests per second is 50 (5*10). In a global rate limiting setup, this limit is shared between all Envoy instances, so the total number of allowed requests per second is 10. 
```

Comment:
IBM Style would say to use the multiplication sign with a space before or after. 

```
(5 × 10)
```

Or maybe just put it in ticks:
```
(`5*10`)
```